test/config/utility.cc

```diff
@@ -313,16 +313,23 @@ std::string ConfigHelper::discoveredClustersBootstrap(const std::string& api_typ
 }
 
 // TODO(#6327) cleaner approach to testing with static config.
-std::string ConfigHelper::adsBootstrap(const std::string& api_type) {
-  return fmt::format(
-      R"EOF(
+// TODO(fredlas) set_node_on_first_message_only was true; the delta+SotW unification
+//               work restores it here. (moved from ads_integration.h)
+std::string ConfigHelper::adsBootstrap(const std::string& api_type,
+                                       envoy::config::core::v3::ApiVersion api_version) {
+  // Note: do not use CONSTRUCT_ON_FIRST_USE here!
+  return fmt::format(R"EOF(
 dynamic_resources:
   lds_config:
+    resource_api_version: {1}
     ads: {{}}
   cds_config:
+    resource_api_version: {1}
     ads: {{}}
   ads_config:
-    api_type: {}
+    transport_api_version: {1}
+    api_type: {0}
+    set_node_on_first_message_only: false
```

Comment:
This technically changes the nature of the base test config, but if no tests fail then it can't be a real dependency. What happens if you leave this as its default? The reason I ask is that this will be very verbose in debug/trace mode.

Reply:
It still passes `bazel test //test/integration/ads_integration_test` when I remove it